### ISSUE TITLE
[Bug?] Failed to refresh profile with gh data: timeout of 30000ms exceeded

### ISSUE THREAD

**Issue by sarpik** [sourcerer-io/sourcerer-app#558]
I tried to `Refresh` > `Update Github repos` @ https://sourcerer.io/sarpik.

I get this error in my console:

```console
baeee485d24a395e21be.js:1 failed to refresh profile with gh data:  Error: timeout of 30000ms exceeded
    at t.exports (e005c28999e832bcba4a.js:2)
    at XMLHttpRequest.d.ontimeout (e005c28999e832bcba4a.js:2)
```

Thanks!

**Comment by yaronskaya** [comment]
Hi @sarpik, could you please check there is an access for Sourcerer in github settings
 `Settings -> Applications -> Authorized OAuth Apps`?

**Comment by sarpik** [comment]
Hello @yaronskaya, thanks, yes there is: https://i.imgur.com/vS0Hzmx.png (under `Authorized OAuth Apps`)

**Comment by yaronskaya** [comment]
@sarpik looks like your github token doesn't provide access for Sourcerer now.
Could you please revoke it and add github account at sourcerer.io again: 
`Settings->Applications->Github: sign in`?

**Comment by sarpik** [comment]
@yaronskaya yep, that did sync the profile.

Though if I click `Refresh` > `Update Github repos`, I still get the same errors and cannot refresh.

Also, before that, I tried signing in with a different browser with the same github account, and a different profile was created - https://sourcerer.io/sarpik1. I don't know if this is expected so just letting you know:D
And they ([sarpik](https://sourcerer.io/sarpik1) and [sarpik1](https://sourcerer.io/sarpik1)) are a little out of sync lol

**Comment by yaronskaya** [comment]
@sarpik yes, these two profiles have different emails. Therefore assigned commits are different. 
Meanwhile, I reassigned the github token from your second profile to the original one - it processes the profile https://sourcerer.io/sarpik now.  Did you change the primary email in github settings?

**Comment by sarpik** [comment]
@yaronskaya I have changed my primary email a while back, but not after I started using sourcerer.

Now the refresh feature seems to work, at least with `/sarpik`:)

Would it be possible to merge the contributions from both (all) of my emails (that my profile has) into a single sourcerer profile, just like github does?
I don't think that having 2 (or more) separate profiles for every email is useful if that's still the same user (at least in my case).

Thanks!

**Comment by yaronskaya** [comment]
@sarpik you can add emails in `Settings` -> `Emails` and after that `Update Github repos`.

**Comment by sarpik** [comment]
Thanks @yaronskaya - I've removed the duplicate account `/sarpik1` and added emails to the main account `/sarpik`. Everything seems to be fine now.

The Refresh feature is now working as expected, some maybe there were some issues with the duplicate account - I don't know.
I suppose we can close this?

Thanks again!

**Comment by kernc** [comment]
[Closing the Sourcerer account](https://sourcerer.io/settings#account) and then reconnecting seems to work.
